.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.8, 3.9]
-        os: [ubuntu-latest, macOS-latest]
+        os: [ubuntu-latest]
```

Comment:
why remove this? seems good to have considering we all develop on Macs

Reply:
To be more efficient with test resources, mac is more expensive and since we're all developing on mac it seems ok to skip running CI on it

Reply:
are we running out of action compute time? I thought it was unlimited if you had a public repo. or you just don't like waiting for how long it takes to run all the tests? just curious

Reply:
If we support mac and windows, we probably should run automated tests on mac and windows, especially if binary data is involved, or python packages that are compiled to c++.  This is part of what gh-actions was made for.  Also why not python 3.10 etc?   Also this is out of scope for this PR and has gotten way more attention than it should :)